packages/amplify-graphql-api-construct/src/amplify-graphql-api.ts

```diff
@@ -211,9 +220,34 @@ export class AmplifyGraphqlApi extends Construct {
       output.payload.awsAppsyncApiKey = this.resources.cfnResources.cfnApiKey.attrApiKey;
     }
 
+    const additionalAuthTypes = this.getAdditionalAuthenticationTypes(this.resources.cfnResources.cfnGraphqlApi);
+    if (additionalAuthTypes) {
+      output.payload.awsAppsyncAdditionalAuthenticationTypes = additionalAuthTypes;
+    }
+
+    if (this.conflictResolution?.project?.handlerType) {
+      output.payload.awsAppsyncConflictResolutionMode = this.conflictResolution?.project?.handlerType;
+    }
+
     outputStorageStrategy.addBackendOutputEntry(graphqlOutputKey, output);
   }
 
+  /**
+   * Transforms additionalAuthenticationTypes for storage in CFN output
+   */
+  private getAdditionalAuthenticationTypes(cfnGraphqlApi: CfnGraphQLApi): string | undefined {
```

Comment:
What's the use-case to need to know the additionalAuthTypes and not the primary auth type?

Reply:
In general, I'd like to keep this transformation logic scoped to the `internal` directory. Could you move this method over to the [authorization-modes.ts](https://github.com/aws-amplify/amplify-category-api/blob/main/packages/amplify-graphql-api-construct/src/internal/authorization-modes.ts) file and add some unit tests to confirm the shape here? Since we're serializing a list I just want to make sure we have clear examples on what the valid string values there are going to be.

Reply:
additionalAuthTypes is used by CMS page in the console. DM'd you with more information.